src/ghost-content/index.js

```diff
@@ -225,7 +225,7 @@ module.exports = ({ logger, db, projectLocation, enabled }) => {
   const readFile = async (rootFolder, file) => {
     const knex = await db.get()
     const { normalizedFolderName } = normalizeFolder(rootFolder)
-    const { isBinary } = folderOptions[normalizedFolderName]
+    const { isBinary } = folderOptions[normalizedFolderName] || {}
```

Comment:
This is actually irrelevant, but I was getting `can't read isBinary of undefined` so I thought it makes sense to fix it too.